packages/flutter/test/painting/text_style_test.dart

```diff
@@ -461,5 +489,10 @@ void main() {
     expect(style.apply(fontFeatures: const <ui.FontFeature>[ui.FontFeature.enable('test')]).fontFeatures, const <ui.FontFeature>[ui.FontFeature.enable('test')]);
     expect(style.apply().textBaseline, TextBaseline.alphabetic);
     expect(style.apply(textBaseline: TextBaseline.ideographic).textBaseline, TextBaseline.ideographic);
+    expect(style.apply().leadingDistribution, TextLeadingDistribution.even);
+    expect(
+      style.apply(leadingDistribution: TextLeadingDistribution.proportional).leadingDistribution,
+      TextLeadingDistribution.proportional,
+    );
```

Comment:
Can you actually test the effect of leadingDistribution in this PR?

Reply:
I tested that in the engine repo but doing basic e2e tests sounds like a good idea!

Reply:
Oh that's right 👍